### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by Ali-0111**
### Learning objectives

1. Use basic JavaScript syntax.
2. Use JavaScript to manipulate DOM elements.
3. Use JavaScript events.

### 
**Project requirements**

1. When the user clicks (or taps) the hamburger button, the mobile menu appears.
2. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
3. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
4. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Ali-0111 and @baezluise, for this project you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository, you are missing the javascript linter config

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that  does not have the javascript linters setup, both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be setup in this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const humberger = document.querySelector('.humberger');
+const mainHead = document.querySelector('.main-head-desktop');
+const spanHide = document.querySelector('.humberger .hide');
+const span1 = document.querySelector('.humberger .span-1');
+const span2 = document.querySelector('.humberger .span-2');
+const links = document.querySelectorAll('nav .flex a');
+
+// munu-tuggle function
+function menuTuggle() {
+  humberger.classList.toggle('p-fixed');
```
Comment (Kossi-stack) [inline app.js:10]: - [ ] When i click the hamburger button, the mobile menu does not appear, kindly fix that.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Ali-0111,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.js (hunk 1 above)
Comment (Ali-0111) [reply app.js:10]: Please manage a zoom meeting, on 10:15pm utc-6. i will be eager to meet you.

**Review by bimbolabuari — COMMENTED** [review]
Hello @Ali-0111  👋,

_This is an additional review as a result of the second opinion request._

- Your mobile menu works as expected 👍 

https://user-images.githubusercontent.com/66112760/217316146-ccad9e58-32ee-412b-89ac-1c68d3005db2.mov


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by osamaashraf6 — APPROVED** [review]

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
.

**Review by osamaashraf6 — COMMENTED** [review]
.

**Review by osamaashraf6 — COMMENTED** [review]
.

**Review by osamaashraf6 — APPROVED** [review]
Hi @Ali-0111 , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Descriptive pull request message ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️

## Optional Suggestion 🎉

- [ ]  You didn't 😃 add the linter js at the project, so for the next milestone, if they will not have the correct linters setup, they will receive it straight away invalidation, so please remember to add it😊

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**